awscli/botocore/args.py

```diff
@@ -196,13 +202,33 @@ def compute_client_args(self, service_model, client_config,
             'service_name': service_name,
             'parameter_validation': parameter_validation,
             'user_agent': user_agent,
+            'configured_endpoint_url': configured_endpoint_url,
             'endpoint_config': endpoint_config,
             'protocol': protocol,
             'config_kwargs': config_kwargs,
             's3_config': s3_config,
             'socket_options': self._compute_socket_options(scoped_config)
         }
 
+    def _compute_configured_endpoint_url(self, client_config, endpoint_url):
+        if endpoint_url is not None or self._ignore_configured_endpoint_urls(
+            client_config
+        ):
+           return endpoint_url
```

Comment:
It looks like we are missing the debug line that got added toward the end of the botocore PR: https://github.com/boto/botocore/pull/2973/files#diff-019afcd91cca6d46dd1046132b61e78198063592925d01e9a4db1fd9f1bf8c3fR281